apps/web/components/team/TeamListItem.tsx

```diff
@@ -126,10 +132,9 @@ export default function TeamListItem(props: Props) {
                       <Link href={"/settings/teams/" + team.id}>
```

Comment:
we should probably use a component here to DRY

Reply:
Sure, checking with @zomars JIC. I can create a follow up PR for it